lib/daru/date_time/index.rb

```diff
@@ -366,6 +366,24 @@ def [] *key
       end
     end
 
+    def pos *args
+      # to filled
+      out = self[*args]
+      return out if out.is_a? Numeric
+      out.map { |date| self[date] }
+    end
+
+    def subset *args
+      self[*args]
+    end
+
+    def respond? *args
```

Comment:
A name that would describe its intent will be `respond_to_index_or_positions?` but its big.